pkg/kv/kvserver/replica_raft.go

```diff
@@ -612,6 +612,11 @@ func (r *Replica) stepRaftGroupRaftMuLocked(req *kvserverpb.RaftMessageRequest)
 	r.raftMu.AssertHeld()
 	var sideChannelInfo replica_rac2.SideChannelInfoUsingRaftMessageRequest
 	err := r.withRaftGroup(func(raftGroup *raft.RawNode) (bool, error) {
+		// If this message requested tracing, begin tracing it.
+		for _, e := range req.TracedEntries {
+			r.mu.raftTracer.RegisterRemote(r.raftCtx, e.Index, e.TraceID, e.SpanID)
```

Comment:
`raftCtx` is created once, when `Replica` is created. Consider passing to the `raftTracer` init method, instead of passing it to each register call.

Also, consider passing the entire `e` to this call, since you're passing all the fields anyway.

Reply:
I didn't end up using this ctx, this was a vestige of the previous approach where I was attempting to make a trace, so its not needed anymore. I also changed to pass the `e`.